core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -155,11 +155,9 @@
                                    gracefulShutdownQuietPeriod, "gracefulShutdownQuietPeriod");
 
         requireNonNull(blockingTaskExecutor, "blockingTaskExecutor");
-        if (!ExecutorServiceMetrics.isMonitoredExecutor(blockingTaskExecutor)) {
```

Comment:
Could you revive this check only for `TimedScheduledExecutorService, TimedExecutorService and TimedExecutor`?
https://github.com/line/armeria/pull/2612#discussion_r395126261

Reply:
Hmm? Any thoughts on what I wrote in the PR description?

Reply:
Oops, I missed the PR description. 😅 That makes sense!